benchmarks/other_benchmarks/startup/benchmark.py

```diff
@@ -0,0 +1,7 @@
+from ...utils import bench_setup
+from django.db import models
+
+class Startup:
+
+    def time_startup(self):
+        bench_setup()
```

Comment:
I think this is only going to test the time of python's `try` and `pass`? Django will already be "setup" by this point so we're going down the exception route 🤔 


https://github.com/smithdc1/django-asv/blob/59c053ae69863dc035dcb593d9435e28315f9af2/benchmarks/utils.py#L9